FABulous/fabric_definition/Fabric.py

```diff
@@ -466,22 +470,23 @@ def determine_border_side(self, x: int, y: int) -> Side | None:
 
         return None
 
-    def get_unique_tile_types(self) -> list[Tile]:
+    def get_all_unique_tiles(self) -> list[Tile | SuperTile]:
```

Comment:
This is a breaking change and should not be buried in such a huge PR, since it might bring changes to other places in the code and should be traceable to a single commit.